flow/project.py

```diff
@@ -370,13 +370,16 @@ class SubmissionJobOperation(JobOperation):
     :type eligible_operations:
         list
     :param operations_with_unmet_preconditions:
-        A list of :py:class:`JobOperation` that will not be executed due to
-        unmet preconditions.
+        A list of :py:class:`JobOperation` that will not be executed in the
+        first pass of :meth:`FlowProject.run` due to unmet preconditions. These
+        operations may be executed in subsequent iterations of the run loop.
     :type operations_with_unmet_preconditions:
         list
     :param operations_with_met_postconditions:
-        A list of :py:class:`JobOperation` that will not be executed due to
-        all postconditions being met.
+        A list of :py:class:`JobOperation` that will not be executed in the
+        first pass of :meth:`FlowProject.run` because all postconditions are
```

Comment:
```suggestion
        current iteration of the :meth:`FlowProject.run` loop because all postconditions are
```

Reply:
See response to the comment for unmet preconditions.